lib/libfrr.c

```diff
@@ -1474,3 +1474,11 @@ const char *frr_vers2str(uint32_t version, char *buf, int buflen)
 
 	return buf;
 }
+
+extern bool frr_is_daemon(void)
```

Comment:
some compilers may not like that "extern" in the definition?

Reply:
fixed